unity-environment/Assets/ML-Agents/Editor/MLAgentsEditModeTest.cs

```diff
@@ -62,7 +62,7 @@ public override void AgentOnDone()
 
     public class TestBrain : Brain
     {
-        // TODO : Mock a brain
+        // Mock a brain
```

Comment:
Move comment to top of class with an explanation as to what purpose the class serves (hint: mock)